sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/aad/implementation/AadOauth2RestOperationConfigurationTestUtil.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.autoconfigure.aad.implementation;
+
+import org.springframework.http.converter.FormHttpMessageConverter;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.security.oauth2.client.http.OAuth2ErrorResponseErrorHandler;
+import org.springframework.security.oauth2.core.http.converter.OAuth2AccessTokenResponseHttpMessageConverter;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.List;
+
+public class AadOauth2RestOperationConfigurationTestUtil {
+
+
+    static boolean hasOAuth2ErrorResponseErrorHandler(RestTemplate restTemplate) {
+        return restTemplate.getErrorHandler() instanceof OAuth2ErrorResponseErrorHandler;
+    }
```

Comment:
We don't need this static method. Inlining this in your method will make it more readable.